packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -101,28 +111,312 @@ export class JobService {
     private readonly storageService: StorageService,
   ) {}
 
+  private async createCvatManifest(dto: JobCvatDto, requestType: JobRequestType, tokenFundAmount: number): Promise<CvatManifestDto> {
+    return {
+      data: {
+        data_url: dto.dataUrl,
+      },
+      annotation: {
+        labels: dto.labels.map((item) => ({ name: item })),
+        description: dto.requesterDescription,
+        user_guide: dto.userGuide,
+        type: requestType,
+        job_size: Number(
+          this.configService.get<number>(ConfigNames.CVAT_JOB_SIZE)!,
+        ),
+        max_time: Number(
+          this.configService.get<number>(ConfigNames.CVAT_MAX_TIME)!,
+        ),
+      },
+      validation: {
+        min_quality: dto.minQuality,
+        val_size: Number(
+          this.configService.get<number>(ConfigNames.CVAT_VAL_SIZE)!,
+        ),
+        gt_url: dto.gtUrl,
+      },
+      job_bounty: await this.calculateJobBounty(dto.dataUrl, tokenFundAmount),
+    };
+  }
+
+  async createHCaptchaManifest(jobType: JobCaptchaShapeType, jobDto: JobCaptchaDto): Promise<HCaptchaManifestDto> {
+    const objectsInBucket = await this.storageService.listObjectsInBucket(jobDto.dataUrl);
+
+    const commonManifestProperties = {
+        job_mode: JobCaptchaMode.BATCH,
+        requester_accuracy_target: jobDto.accuracyTarget,
+        request_config: {},
+        requester_max_repeats: jobDto.maxRequests,
+        requester_min_repeats: jobDto.minRequests,
+        requester_question: { en: jobDto.annotations.labelingPrompt },
+        job_total_tasks: objectsInBucket.length,
+        task_bid_price: jobDto.annotations.taskBidPrice,
+        taskdata_uri: await this.generateAndUploadTaskData(jobDto.dataUrl, objectsInBucket),
+        public_results: true,
+        oracle_stake: 0.05,
```

Comment:
why is this value constant?

Reply:
I will clarify whether this field is dynamic or static.

Reply:
@eugenvoronov have clarified this?

Reply:
@portuu3 Still didn't get response, I pinged them again.